_includes/templates/download.html

```diff
@@ -87,7 +86,11 @@ <h2>{{ page.latestversion }} {{CURRENT_RELEASE}} <a type="application/rss+xml" h
   <h2 style="text-align: center">{{ page.patient }}</h2>
   <p>{{ page.notesync | replace: '$(DATADIR_SIZE)', site.data.stats.datadir_gb | replace: '$(PRUNED_SIZE)', site.data.stats.pruned_gb | replace: '$(MONTHLY_RANGE_GB)', site.data.stats.monthly_storage_increase_range_gb }} {{ page.full_node_guide }}</p>
 
-{% if page.version > 4 %}
+
+  <h2 style="text-align: center">{{ page.verify_title }}</h2>
+  <p>{{ page.verify_steps }}</p>
+
+{% if page.version > 1 %}
```

Comment:
Let's leave this as `page.version > 4` and edit `_posts/en/pages/2017-01-01-download.md` to `version: 5`.  This will hide the updated instructions on the Japanese translation until the updated strings can be translated.

Reply:
Yup, I like your suggestion much better.